deploy/alerts/simple-comparison.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: fixed-compares
+    role: alert-rules
+  name: prometheus-alarms-part-two
```

Comment:
`name: stf-default-prometheus-openstack-rules-simple`? Not sure what to suggest as I don't know why these can't be merged together.

Reply:
See above, it's mostly how they became generated.

Reply:
The above comment still doesn't really explain why there actually needs to be two separate files other than that's how your script was created. When providing this to a customer it's not going to be obvious why there are two separate files, and what to adjust should they want to customize things based on this example.

Reply:
There is no technical need to have it in separate files. It is more convenient to handle though. 
I've tried to add some more expanation in https://github.com/infrawatch/service-telemetry-operator/pull/76/files , deploy/alerts/README.md